core/hotspot/rule_manager.go

```diff
@@ -219,6 +223,16 @@ func onRuleUpdate(rules []*Rule) (err error) {
 			}
 		}
 	}
+
+	tcMux.Lock()
+	defer func() {
+		tcMux.Unlock()
+		if r := recover(); r != nil {
+			return
+		}
```

Comment:
Seems no need to do recover at here, as there are only some very simple assignment operations.

Reply:
Done, thanks.